Dockerfile

```diff
@@ -53,8 +53,8 @@ RUN apk add --update musl go=$GO_VERSION git bash wget ca-certificates \
 	# We get the current commit using this hack,
 	# so that we don't have to copy all of .git/ into the build context.
 	# This saves us quite a bit of image size.
-	&& ref="$(cat .git/HEAD | cut -d' ' -f2)" \
-	&& commit="$(cat .git/$ref | head -c 7)" \
+	&& ref=$(cat .git/HEAD | grep ref | cut -d' ' -f2) \
+	&& commit=$(if [ -z "$ref" ]; then cat .git/HEAD; else cat ".git/$ref"; fi | head -c 7) \
```

Comment:
Hey thanks for wighing in! -- do you happen to know if `rev-parse` and `symbolic-ref` work without an actual repo, i.e. with only `.git/HEAD` and `.git/refs` or something similar present? The idea with this hack is that we don't have to copy all of `.git`, and thus gain savings in image size.